osquery/core/CMakeLists.txt

```diff
@@ -12,7 +12,7 @@ file(GLOB OSQUERY_CORE_PLATFORM_TESTS "tests/${PROCESS_FAMILY}/*.cpp")
 if(APPLE)
   set(OSQUERY_CORE_PLATFORM
     ${OSQUERY_CORE_PLATFORM}
-    darwin/conversions.cpp
+    darwin/conversions_darwin.cpp
```

Comment:
Can you describe the problem you're trying to fix here?

Reply:
There were two `conversions.cpp` files and object files generated for them were `conversions-<random_unique_suffix>.o` but they were referenced on the link stage as `conversions.o` with did not exist for any of the two (probably a cmake issue)…